spec/models/nonprofit_spec.rb

```diff
@@ -143,11 +143,22 @@
         expect(nonprofit_with_bad_email_and_website.errors['email'].first).to match /.*invalid.*/ 
       end
 
-      it 'marks website as having errors if they do' do
-        expect(nonprofit_with_bad_email_and_website.errors['website'].first)
-          .to match('is not a valid URL')
+      context 'when the website has errors' do
+        it 'marks as having weeoes if they don not have a public suffix' do
```

Comment:
What's the title of this spec? :smile:

Reply:
hahaha it should be 'errors' instead of weeoes, I don't know what happened :laughing: